### ISSUE TITLE
Several issues with Australia

### ISSUE THREAD

**Issue by arandomperson5000** [simsulla/nwomod#806]
Looked at Australia in the mod and I noticed a couple of things that made me to start thinking about.
1. I don't get why Sydney is the capital of Australia in this mod even though Canberra is the capital of                       Australia, does this have to do with pop migration and region issues or does this stem from the common misconception that either Melbourne or Sydney is the capital of Australia.
2. This point is more a general question than anything else really but is it realistic to put in Western Australia as a country in this mod as it was quite strong as a movement in throughout the 20th century. In 1933 an official referendum was held in Western Australia referendum on secession alongside the State parliamentary election. 68% of the 237,198 voters voted in favour of secession however the British Parliament due to them deciding that the request could not be acted upon because it did not have the support of the Australian federal government, as required by the 1931 Statute of Westminster. The movement faded into obscurity until 1974 when the iron ore magnate Lang Hancock founded the  Westralian Secession Movement which unsuccessfully tried to put a candidate in the 1974 Senate election. Aside from a few studies and controversial comments from several politicians it appears the movement is a little quiet nowadays.
3. I think there could a potential for decisions to create new Australian States out of places like Papua New Guinea, New Zealand has a particularly strong case to join Australia as two separate states as there were several suggestions and proposals in the past for New Zealand to join Australia, I think the original Australian Constitution actually recognises New Zealand as an actual state. Also during the process of Portuguese decolonisation in East Timor, a political party was formed called ADITLA Associação Democrática para a Integração de Timor Leste na Austrália (Democratic Association for the Integration of East Timor into Australia) by local businessman Henrique Pereira. It found some support from the ethnic Chinese community, fearful of independence or integration with Indonesia but was disbanded when the Australian government rejected the idea in 1975.

Sources
http://www.sro.wa.gov.au/blogs/westralia-shall-be-free-western-australian-secession-referendum-1933
https://www.elections.wa.gov.au/elections/referendums/past-referendums/sr1933
https://www.youtube.com/watch?v=F8C-2PvhmX8
http://easttimorgovernment.com/history.htm
https://books.google.co.uk/books?id=8uxwaCR_71cC&pg=PA58&lpg=PA58&dq=Aditla&source=web&ots=w_LDdYRH-t&sig=h6N8Vgm6ZyGidn1sINs88CiaHFE&hl=en#v=onepage&q=Aditla&f=false
http://www.nzhistory.net.nz/page/nz-says-no-aussie-federation
https://www.theguardian.com/world/2015/nov/25/new-zealand-should-become-seventh-and-eighth-states-of-australia-jokes-senator
http://tvnz.co.nz/national-news/full-umr-research-poll-results-aust-nz-union-3414686

**Comment by simsulla** [comment]
I like the suggestions. The [Pacific Islands Forum](https://en.wikipedia.org/wiki/Pacific_Islands_Forum) is in the mod and to some extent mimics what you are suggesting. Canberra is not capital because to have a nice map a new province is needed.

**Comment by arandomperson5000** [comment]
Thanks for letting me the Pacific Islands Forum is in the mod, what I was rather suggesting above is a decision for New Zealand and East Timor that allows Australia to peacefully annex them and creates ores for Australia on those countries, it would be similar to how the United Arab Republic gets bigger every time when an Arab country says yes to an event/decision to join it. As for Papua New Guinea since it was once owned by Australia, the decision would for Australia that allows them to put Australian cores all over New Guinea after a specific date like 1980 or 1990.

**Comment by simsulla** [comment]
Ok, so reading up on these proposals. Secessionists in Western Australia in the 70s gained little traction so not sure it warrants a separate core. Also that Australia integrating East Timor or Papua i believe is a bit to unlikely, except perhaps as part of a larger pacific integration project - that at the moment exists in the mod. For Australia to keep Papua was not an options as it was a United Nations Trust Territory, and thus legally not theirs. To my knowledge there is no Papuan party with significant support advocating joining Australia. 

TO sum it up the The Pacific Islands Forum covers this form a player perspective as Australia wanting to build an empire. Will close this issue.
